test/test_open.py

```diff
@@ -12,6 +13,23 @@
 FILENAME = Path('one')
 
 
+# Windows OS helper for checking admin privileges
```

Comment:
Quibble: lose the comment!  The name of the function is good enough, and it's only a test!